### PR TITLE
Release 1.13.0

### PR DESCRIPTION
**Pull request by vrubezhny**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+## 1.13.0 (April 10, 2024)
+
+ * [#4045](https://github.com/redhat-developer/vscode-openshift-tools/issues/4045) Can't start crc via the extension.
```
Comment (datho7561) [inline CHANGELOG.md:5]: Rename to "fix starting crc via the extension"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+ * [#4013](https://github.com/redhat-developer/vscode-openshift-tools/issues/4013) Provide an option to delete the Pod like k8s.
+ * [#4004](https://github.com/redhat-developer/vscode-openshift-tools/issues/4004) Survey doesn't work.
```
Comment (datho7561) [inline CHANGELOG.md:7]: Rename to "fix survey"

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,16 @@
...
+ * [#3991](https://github.com/redhat-developer/vscode-openshift-tools/issues/3991) Re-arrange application explorer similar to vsKubernetes cluster view.
+ * [#3981](https://github.com/redhat-developer/vscode-openshift-tools/issues/3981) Migrate Functionality from Kubernetes VSCode Extension to OpenShift Toolkit.
+ * [#3947](https://github.com/redhat-developer/vscode-openshift-tools/issues/3947) The App Explorer tree isn't refreshed after logging in to a Kind cluster.
+ * [#3919](https://github.com/redhat-developer/vscode-openshift-tools/issues/3919) Improve "Open Kuberbetes YAML file" to avoid accidental opens and duplicate files.
+ * [#3918](https://github.com/redhat-developer/vscode-openshift-tools/issues/3918) Fix Action labels for kubernetes clusters.
```
Comment (datho7561) [inline CHANGELOG.md:12]: I think this should be more specific, eg. "Use "namespaces" wording on kubernetes clusters and "projects" wording on OpenShift clusters"

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+## 1.13.0 (April 10, 2024)
+
+ * [#4045](https://github.com/redhat-developer/vscode-openshift-tools/issues/4045) Can't start crc via the extension.
+ * [#4013](https://github.com/redhat-developer/vscode-openshift-tools/issues/4013) Provide an option to delete the Pod like k8s.
+ * [#4004](https://github.com/redhat-developer/vscode-openshift-tools/issues/4004) Survey doesn't work.
+ * [#3991](https://github.com/redhat-developer/vscode-openshift-tools/issues/3991) Re-arrange application explorer similar to vsKubernetes cluster view.
+ * [#3981](https://github.com/redhat-developer/vscode-openshift-tools/issues/3981) Migrate Functionality from Kubernetes VSCode Extension to OpenShift Toolkit.
+ * [#3947](https://github.com/redhat-developer/vscode-openshift-tools/issues/3947) The App Explorer tree isn't refreshed after logging in to a Kind cluster.
```
[the remark this answers is not in the document]
Comment (vrubezhny) [reply CHANGELOG.md:10]: ACK

File: CHANGELOG.md (hunk 1 above)
Comment (mohitsuman) [inline CHANGELOG.md:5]:
```suggestion
 * [#4045](https://github.com/redhat-developer/vscode-openshift-tools/issues/4045) Bump minimum memory for CRC to 10752 MB
```

**Review by datho7561 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (vrubezhny) [reply CHANGELOG.md:7]: ACK

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+## 1.13.0 (April 10, 2024)
+
+ * [#4045](https://github.com/redhat-developer/vscode-openshift-tools/issues/4045) Can't start crc via the extension.
+ * [#4013](https://github.com/redhat-developer/vscode-openshift-tools/issues/4013) Provide an option to delete the Pod like k8s.
```
Comment (mohitsuman) [inline CHANGELOG.md:6]:
```suggestion
 * [#4013](https://github.com/redhat-developer/vscode-openshift-tools/issues/4013) Support deletion of Pod resources from extension UI.
```

File: CHANGELOG.md (hunk 1 above)
Comment (datho7561) [reply CHANGELOG.md:5]: Acutally, use Mohit's suggestion

File: CHANGELOG.md (hunk 2 above)
Comment (datho7561) [reply CHANGELOG.md:7]: Use Mohit's suggestion

File: CHANGELOG.md (hunk 1 above)
Comment (vrubezhny) [reply CHANGELOG.md:5]: ACK

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,16 @@
...
+ * [#4004](https://github.com/redhat-developer/vscode-openshift-tools/issues/4004) Survey doesn't work.
+ * [#3991](https://github.com/redhat-developer/vscode-openshift-tools/issues/3991) Re-arrange application explorer similar to vsKubernetes cluster view.
```
Comment (mohitsuman) [inline CHANGELOG.md:8]:
```suggestion
 * [#3991](https://github.com/redhat-developer/vscode-openshift-tools/issues/3991) Showcase all Workload resources in Application Explorer Tree view.
```

**Review by mohitsuman — CHANGES_REQUESTED** [review]
added wording changes

File: CHANGELOG.md (hunk 3 above)
Comment (vrubezhny) [reply CHANGELOG.md:12]: ACK

File: CHANGELOG.md (hunk 2 above)
Comment (vrubezhny) [reply CHANGELOG.md:7]: Used Mohit's version: `Fix Extension Survey form`

File: CHANGELOG.md (hunk 1 above)
Comment (vrubezhny) [reply CHANGELOG.md:5]: Used Mohit's version for this: "Bump minimum memory for CRC to 10752 MB"

File: CHANGELOG.md (hunk 5 above)
Comment (vrubezhny) [reply CHANGELOG.md:6]: ACK

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+## 1.13.0 (April 10, 2024)
+
+ * [#4045](https://github.com/redhat-developer/vscode-openshift-tools/issues/4045) Bump minimum memory for CRC to 10752 MB
+ * [#4013](https://github.com/redhat-developer/vscode-openshift-tools/issues/4013) Support deletion of Pod resources from extension UI
+ * [#4004](https://github.com/redhat-developer/vscode-openshift-tools/issues/4004) Fix Extension Survey form
+ * [#3991](https://github.com/redhat-developer/vscode-openshift-tools/issues/3991) Showcase all Workload resources in Application Explorer Tree view
+ * [#3981](https://github.com/redhat-developer/vscode-openshift-tools/issues/3981) Migrate Functionality from Kubernetes VSCode Extension to OpenShift Toolkit.
+ * [#3947](https://github.com/redhat-developer/vscode-openshift-tools/issues/3947) The App Explorer tree isn't refreshed after logging into a Kind cluster.
```
[the remark this answers is not in the document]
Comment (vrubezhny) [reply CHANGELOG.md]: ACK

**Review by datho7561 — APPROVED** [review]
Thanks, Victor!

**Pull request merged**